proposals/0005-objective-c-name-translation.md

```diff
@@ -569,8 +569,8 @@ func <b>with</b>Matrix(_: CGAffineTransform) -> UIFontDescriptor
 </pre>
 
 If we instead began by pruning `SymbolicTraits` from the tail of
-the first method name, the prohibition against creating `absurdly
-vacuous names`_ would prevent us from pruning "`fontDescriptorWith`"
```

Comment:
That was supposed to be an internal hyperlink (mis-translated from the original ReST).